scripts/download_process_movieLens_data.py

```diff
@@ -25,10 +25,9 @@
                        'salesman', 'scientist', 'student', 'technician', 'writer']
 MOVIE_FEATURE_VALUES = MOVIE_FEATURE_COLUMNS
 
-DETEXT_LABEL = 'label'
 DETEXT_QUERY = 'doc_query'
-DETEXT_WIDE_IDX = 'wide_ftrs_sp_idx'
-DETEXT_WIDE_VAL = 'wide_ftrs_sp_val'
+DETEXT_WIDE_IDX = 'wide_ftrs_sp_indices0'
```

Comment:
This is new name from DeText?

Reply:
Yes, DeText followed the tensor format we used internally, which are hard coded names, users can't pass customized column names for sparse tensor name and value.